requirements-dev.txt

```diff
@@ -7,7 +7,7 @@ coremltools
 pandas
 pytest
 pytest-cov
-scikit-learn
+scikit-learn==0.19.2
```

Comment:
Same here, I removed the constraint on scikit-learn's version. Is it really needed?